### PR TITLE
feat(multiedit): Add/remove items and panels on multiedit

### PR DESCRIPTION
**Pull request by nidzix**
SD-1488
SD-1504

### CODE REVIEW

**Comment by nidzix** [comment]
@petrjasek @amagdas ?

File: app/scripts/superdesk-authoring/multiedit/views/multiedit.html [hunk 1]
```diff
@@ -2,14 +2,27 @@
 
 <div class="multiedit-screen main-section">
     <div class="boards-list">
-    	<div ng-repeat="board in boards track by board.article"
+    	<div ng-repeat="board in boards track by $index"
```
Comment (petrjasek) [inline app/scripts/superdesk-authoring/multiedit/views/multiedit.html]: I think it should be tracking by article `_id`

File: app/scripts/superdesk-authoring/multiedit/tests/multiedit_spec.js [hunk 2]
```diff
@@ -6,17 +6,25 @@ describe('multiedit', function() {
     it('can open an item', inject(function(multiEdit) {
         var items = multiEdit.items;
         expect(items.length).toBe(0);
-        multiEdit.edit('foo');
+        multiEdit.edit('foo', 0);
         expect(multiEdit.items.length).toBe(1);
-        expect(multiEdit.items).not.toEqual(items);
+        expect(multiEdit.items).toEqual(items);
```
Comment (petrjasek) [inline app/scripts/superdesk-authoring/multiedit/tests/multiedit_spec.js]: why this change?

Comment (amagdas) [inline app/scripts/superdesk-authoring/multiedit/tests/multiedit_spec.js]: Probably there is always an open article unused like in the proposed design ...

**Comment by amagdas** [comment]
Besides @petrjasek comments, :+1:

File: app/scripts/superdesk-authoring/multiedit/views/multiedit.html (hunk 1 above)
Comment (nidzix) [inline app/scripts/superdesk-authoring/multiedit/views/multiedit.html]: I know that it is not a good practice to track by `$index` but we can have empty boards open without any article in it. So there will be duplicates...

File: app/scripts/superdesk-authoring/multiedit/tests/multiedit_spec.js (hunk 2 above)
Comment (nidzix) [inline app/scripts/superdesk-authoring/multiedit/tests/multiedit_spec.js]: Service functions are rewritten. I think you were using `lodash` functions which where returning new array, and I am using now plain js funcitons.

Comment (petrjasek) [inline app/scripts/superdesk-authoring/multiedit/tests/multiedit_spec.js]: yep but I was doing it so there is no need for recursive `$watch`, it was by design..

File: app/scripts/superdesk-authoring/multiedit/views/multiedit.html (hunk 1 above)
Comment (petrjasek) [inline app/scripts/superdesk-authoring/multiedit/views/multiedit.html]: I think it will be better not to have it there then

**Comment by petrjasek** [comment]
:+1:
